src/activity-logger/background/log-page-visit.js

```diff
@@ -52,17 +58,23 @@ export async function logPageVisit({ tabId, url }) {
     // The time to put in documents.
     const timestamp = Date.now()
 
-    // First create an identifier for the page being visited.
-    const storePageResult = await storePage({ tabId, url })
+    const pageId = generatePageDocId({ url })
+    const existingPage = index.initSingleLookup()(pageId)
+
+    let storePageResult
+    if (existingPage == null || timestamp - existingPage.latest < threshold) {
```

Comment:
This `<` should be a `>`. I think this what we talked about a few days back (I think I suggested the `<` though)